multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -147,6 +147,15 @@ def siamese_network():
     return automm_presets.create("default")
 
 
+@automm_presets.register()
+def text_match_text():
```

Comment:
Consider to change to `text_similarity`.

Reply:
Done.